docs/try-it-out/longer-introduction.md

```diff
@@ -4,36 +4,36 @@ description: Quickstart covering key concepts in n8n.
 contentType: tutorial
 ---
 
-# A slightly longer introduction
+# Your first workflow
 
-This guide shows you how to automate a task using a [workflow](/glossary.md#workflow-n8n) in n8n, explaining key concepts along the way. You will:
+This guide will show you how to construct a [workflow](/glossary.md#workflow-n8n) in n8n, explaining key concepts along the way. You will:
 
-* Create a workflow from scratch. You'll build a workflow that runs every week to get data from NASA, filters the data, and generates two reports.
+* Create a workflow from scratch. 
 * Understand key concepts and skills, including:
     * Starting workflows with trigger nodes
     * Configuring [credentials](/glossary.md#credential-n8n)
-    * Manipulating data
+    * Processing data
     * Representing logic in an n8n workflow
     * Using [expressions](/glossary.md#expression-n8n)
 
+This quickstart uses [n8n Cloud](/manage-cloud/overview.md), which is recommended for new users. A free trial is available - if you haven't already done so, [sign up](https://app.n8n.cloud/register) for an account now.
 
-## Step one: Sign up for n8n
+## Step one: Create a new workflow
 
-This quickstart uses [n8n Cloud](/manage-cloud/overview.md). A free trial is available for new users. If you haven't already done so, [sign up](https://app.n8n.cloud/register) for an account now.
+When you open n8n, you'll see either:
 
-## Step two: New workflow
+* An empty workflow: if you have no workflows and you're logging in for the first time. Use this workflow.
```

Comment:
I actually didn't see either of these on the account I created recently. Instead, there was a page with a button to either start from scratch or do something with an AI workflow. (it doesn't seem to drop you onto a blank canvas anymore).

Reply:
Thanks!